engine/glfw/lib/android/android_init.c

```diff
@@ -708,7 +708,6 @@ static int32_t addInputEvents(struct android_app* app, const AInputEvent* event,
         out->m_DownTime = AKeyEvent_getDownTime(event);
         out->m_EventTime = AKeyEvent_getEventTime(event);
         (*out_count)++;
-        out++;
```

Comment:
We were reading the values from the wrong pointer.

Reply:
Oh, yeah, that's not good :-) And this problem was introduced during the engine refactor?